### DISCUSSION TITLE
New Tutorials for Triton Inference Server Now Available

### DISCUSSION THREAD

**Discussion by tanayvarshney** [triton-inference-server/server#5426] [Announcements]
New to Triton Inference Server? Not sure where to begin? Check out [this tutorials repository](https://github.com/triton-inference-server/tutorials) for Triton Inference Server! You will find: 
* A comprehensive guide for onboarding new users to Triton
* A guide about deploying models from HuggingFace
* A quick deploy guide for the most popular frameworks.
readily available! Want a new guide? Open a new issue with "Requested Example: <name>" [here](https://github.com/triton-inference-server/tutorials/issues).

**Comment by saitejamalyala** [comment]
@tanayvarshney ,  following the hf pipeline using python backend for serving whisper on TIS. 

Stuck with this https://github.com/triton-inference-server/server/issues/7847 , any thoughts on why this would happen ?
